src/test/fio/fio_ceph_objectstore.cc

```diff
@@ -209,7 +209,7 @@ Job::Job(Engine* engine, const thread_data* td)
   for (uint32_t i = 0; i < td->o.nr_files; i++) {
     auto f = td->files[i];
     f->real_file_size = file_size;
-    f->engine_data = i;
```

Comment:
thank you both for looking at this. i agree that `engine_pos` is what we want here, as it matches what they did with other engines in https://github.com/axboe/fio/commit/710bf9c592b849935e1bcc56ffc805fa1d5f9365:
```diff
-#define LAST_POS(f)	((f)->engine_data)
+#define LAST_POS(f)	((f)->engine_pos)
```